src/main/kotlin/com/easternkite/remain/Routing.kt

```diff
@@ -42,13 +43,29 @@ fun Application.configureRouting() {
                     text = Json.encodeToString(it),
                     contentType = ContentType.Application.Json
                 )
-            }.onFailure {
-                call.respond(HttpStatusCode.BadRequest)
+            }.onFailure { error ->
+                call.respond("${HttpStatusCode.BadRequest} - ${error.message}")
             }
```

Comment:
요거 혹시 Dooray Hook으로 보낼 목적이라면,
`DrResponse`에 담아서 보내야 정상적으로 메세지 훅으로 보여질 것 같습니다 :blush:

```suggestion
            }.onFailure { error ->
                val errorResponse = DrResponse(
                    text = "${HttpStatusCode.BadRequest} - ${error.message}",
                    responseType = "ephemeral" // 작성자한테만 표시
                )

                call.respondText(
                    text = Json.encodeToString(errorResponse),
                    contentType = ContentType.Application.Json
                )
            }
```

Reply:
오 제가 하고싶던게 이거에요! :+1: